packages/authorization-process/test/getClientKeys.test.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  getMockClient,
+  getMockKey,
+} from "pagopa-interop-commons-test/src/testUtils.js";
+import {
+  Client,
+  Key,
+  UserId,
+  generateId,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  clientNotFound,
+  organizationNotAllowedOnClient,
+} from "../src/model/domain/errors.js";
+import { addOneClient, authorizationService } from "./utils.js";
+
+describe("getClientKeys", async () => {
+  const consumerId = generateId();
+  const mockKey = getMockKey();
+  const mockClient: Client = {
+    ...getMockClient(),
+    keys: [mockKey],
+    consumerId: unsafeBrandId(consumerId),
+  };
+
+  it("should get the keys in the specified client", async () => {
+    const keyUserId1: UserId = generateId();
+    const keyUserId2: UserId = generateId();
+    const keyUserId3: UserId = generateId();
+
+    const keyWithUser1: Key = {
+      ...mockKey,
+      userId: keyUserId1,
+    };
+    const keyWithUser2: Key = {
+      ...mockKey,
+      userId: keyUserId2,
+    };
+    const keyWithUser3: Key = {
+      ...mockKey,
+      userId: keyUserId3,
+    };
+    const clientWithKeyUser: Client = {
+      ...mockClient,
+      keys: [keyWithUser1, keyWithUser2, keyWithUser3],
+      users: [keyUserId1, keyUserId2, keyUserId3],
+    };
+    await addOneClient(clientWithKeyUser);
+
+    const keys = await authorizationService.getClientKeys({
+      clientId: mockClient.id,
+      userIds: [keyUserId1, keyUserId2, keyUserId3],
```

Comment:
Could you exclude at least one `userId` from the query? So that we could ensure the filter is working.